server/src/test/YmlKaoFileVisitor.test.ts

```diff
@@ -198,81 +156,22 @@ describe('Extension Server Tests', () => {
-                            end: {
-                                character: 30,
-                                line: 4,
-                            },
-                            start: {
-                                character: 18,
-                                line: 2,
-                            },
-                        },
-                        uri: '',
-                    },
-                    detail: 'String',
-                    documentation: NOT_DOCUMENTED,
-                    kind: CompletionItemKind.Method,
-                    label: 'getName',
-                    uri: '',
-                },
-                {
-                    data: 'id_City_writeCountry',
-                    definitionLocation: {
-                        range: {
-                            end: {
-                                character: 30,
-                                line: 7,
-                            },
-                            start: {
-                                character: 18,
-                                line: 5,
-                            },
-                        },
-                        uri: '',
-                    },
-                    detail: 'String',
-                    documentation: NOT_DOCUMENTED,
-                    kind: CompletionItemKind.Method,
-                    label: 'writeCountry',
-                    uri: '',
-                },
-                {
-                    data: 'id_City_inhabitants',
-                    definitionLocation: {
-                        range: {
-                            end: {
-                                character: 36,
-                                line: 11,
-                            },
-                            start: {
-                                character: 18,
-                                line: 8,
-                            },
-                        },
-                        uri: '',
-                    },
-                    detail: 'Collection − Person',
-                    documentation: NOT_DOCUMENTED,
-                    kind: CompletionItemKind.Property,
-                    label: 'inhabitants',
-                    uri: '',
-                },
-            ];
-            assert.deepEqual(completionProvider.completions, expectedCompletionItems);
+            assert.strictEqual(completionProvider.completions.length, 4);
+            assertHasNelementsRemaining(
+                completionProvider.completions,
+                1,
+                (elem) => elem.kind === CompletionItemKind.Class,
+            );
+            assertHasNelementsRemaining(
+                completionProvider.completions,
+                1,
+                (elem) => elem.kind === CompletionItemKind.Property,
+            );
+            assertHasNelementsRemaining(
+                completionProvider.completions,
+                2,
+                (elem) => elem.kind === CompletionItemKind.Method,
+            );
```

Comment:
Kinda scary to see that the tests are shorter xD Do you check thinks as thoroughly as before? Is it just some kind of refacto?
Omg the old version looks dirty, actually xD

Reply:
It's clearly less checked as before, but I think that trying to get full equality between objects was very hard to maintain.
Also, the main goal of these tests is mostly to check that we really get completion items and that they are ok-ish.